bin/run-ansible.sh

```diff
@@ -48,6 +50,14 @@ case ${key} in
 esac
 done
 
+if [[ "${playbook}" = "server_and_nuc_deploy" ]]; then
+    if [[ -z ${read_secure} ]]; then
+        read_secure="sudo_password"
+    else
+        read_secure=$read_secure+",sudo_password"
```

Comment:
why dont you simply have this in both occasions?

Reply:
because it creates a comma